python/samples/kernel-syntax-examples/chat_gpt_api_function_calling.py

```diff
@@ -77,17 +75,6 @@
     max_auto_invoke_attempts=3,
 )
 
-prompt_template_config = sk.PromptTemplateConfig(
-    template="{{$user_input}}",
-    name="chat",
-    template_format="semantic-kernel",
-    input_variables=[
-        InputVariable(name="user_input", description="The user input", is_required=True),
```

Comment:
Curious why we want to remove showing how to configure the PromptTemplateConfig with the input_variables? Yeah, the proposed change is less code, but isn't it important to show how this can be done?

Reply:
You are correct, we need to look at the full range off samples and decide where to fully explain what, I can add these back in.